test_model_configs/single_population_node_with_updaters_config.txt

```diff
@@ -0,0 +1,732 @@
+index: 0
+name: "root"
+branch_node {
+  branches {
+    condition {
+      name: "cookie vs. cookieless. name field is optional."
+      op: EQUAL
+      value: "cookied"
+    }
+    node {
+        name: "cookied"
+        index: 2
+        branch_node {
+          branches {
+            condition {
+              name: "pool"
+              op: EQUAL
+              value: "cookied_pool"
+            }
+            node {
+              name: "cookied_pool"
+              index: 0
+                branch_node {
+                  branches {
+                    condition {
+                      name: "country"
```

Comment:
The name points to a proto field. Here it is "labeler_input.geo.country_id". See LabelerEvent in model.proto.
Note that it is an int. 
Alternatively, if earlier in the model we compute "person_country_code" from "labeler_input.geo.country_id", here we may use "person_country_code".